src/rf/apps/workers/process.py

```diff
@@ -28,6 +28,14 @@
 DEFAULT_DELAY = 0
 TIMEOUT_DELAY = 60
 
+# Error messages
+ERROR_MESSAGE_LAYER_IMAGE_INVALID = 'Cannot process invalid images.'
+ERROR_MESSAGE_IMAGE_NOT_VALID = 'Image was not a valid GeoTiff file.'
+ERROR_MESSAGE_IMAGE_TOO_FEW_BANDS = 'Image must have three or more bands.'
+ERROR_MESSAGE_JOB_TIMEOUT = 'Layer could not be processed. ' + \
+                            'The job timed out after ' + \
+                            str(TIMEOUT_SECONDS / 60) + ' minutes.'
```

Comment:
Good idea. Actually I'd rather just make it an integer. Round it up to the nearest minute. No one cares if it took .5 or 1 minute.